#!/usr/bin/env python
PACKAGE = "sonar_image_processing"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("blur_size", int_t, 0, "Blurring window size.", 5, 0, 10)
gen.add("threshold_size", int_t, 1, "Filtering threshold.", 75, 0, 100)
gen.add("threshold_offset", int_t, 2, "Threshold offset.", 25, 0, 100)
gen.add("target_size", int_t, 4, "Size of the target.", 0, 0, 2000000)

exit(gen.generate(PACKAGE, "sonar_image_processing", "SonarDetector"))
